dm/tests/dmctl_command/run.sh

```diff
@@ -152,6 +152,261 @@ function run() {
 	fi
 }
 
+function run_validation_start_stop_cmd {
+	cleanup_data dmctl_command
+	cleanup_data_upstream dmctl_command
+	cleanup_process $*
+
+	export GO_FAILPOINTS=""
+	run_dm_master $WORK_DIR/master $MASTER_PORT $cur/conf/dm-master.toml
+	check_rpc_alive $cur/../bin/check_master_online 127.0.0.1:$MASTER_PORT
+	run_dm_worker $WORK_DIR/worker1 $WORKER1_PORT $cur/conf/dm-worker1.toml
+	check_rpc_alive $cur/../bin/check_worker_online 127.0.0.1:$WORKER1_PORT
+	run_dm_worker $WORK_DIR/worker2 $WORKER2_PORT $cur/conf/dm-worker2.toml
+	check_rpc_alive $cur/../bin/check_worker_online 127.0.0.1:$WORKER2_PORT
+
+	run_sql_file $cur/data/db1.prepare.sql $MYSQL_HOST1 $MYSQL_PORT1 $MYSQL_PASSWORD1
+	run_sql_file $cur/data/db2.prepare.sql $MYSQL_HOST2 $MYSQL_PORT2 $MYSQL_PASSWORD2
+
+	dmctl_operate_source create $cur/conf/source1.yaml $SOURCE_ID1
+	dmctl_operate_source create $cur/conf/source2.yaml $SOURCE_ID2
+	dmctl_start_task $cur/conf/dm-task-no-validator.yaml --remove-meta
+	run_dm_ctl_with_retry $WORK_DIR "127.0.0.1:$MASTER_PORT" \
+		"query-status test" \
+		"\"unit\": \"Sync\"" 2
+
+	echo "--> (fail) validation start: missing mode value"
+	run_dm_ctl $WORK_DIR "127.0.0.1:$MASTER_PORT" \
+		"validation start --mode" \
+		"Error: flag needs an argument: --mode" 1
+
+	echo "--> (fail) validation start: invalid mode"
+	run_dm_ctl $WORK_DIR "127.0.0.1:$MASTER_PORT" \
+		"validation start --mode xxx" \
+		"Error: mode should be either \`full\` or \`fast\`" 1
+
+	echo "--> (fail) validation start: missing start-time value"
+	run_dm_ctl $WORK_DIR "127.0.0.1:$MASTER_PORT" \
+		"validation start --start-time" \
+		"Error: flag needs an argument: --start-time" 1
+
+	echo "--> (fail) validation start: invalid start-time"
+	run_dm_ctl $WORK_DIR "127.0.0.1:$MASTER_PORT" \
+		"validation start --start-time xx" \
+		"Error: start-time should be in the format like '2006-01-02 15:04:05' or '2006-01-02T15:04:05'" 1
+
+	echo "--> (fail) validation start: without all-task and task-name"
+	run_dm_ctl $WORK_DIR "127.0.0.1:$MASTER_PORT" \
```

Comment:
What about moving these failed cases to `run_validator_error_cmd`?

Reply:
some negative case depends on positive case, so it's mixed together, so it's separate with other commands